solr/Dockerfile.okd

```diff
@@ -0,0 +1,59 @@
+FROM solr:7.7.2-slim
+
+#COPY --chown=solr:solr config /opt/solr/server/solr/persons
+#COPY --chown=solr:solr config /opt/solr/server/solr/organizations
+#COPY --chown=solr:solr config /opt/solr/server/solr/documents
+#COPY --chown=solr:solr config /opt/solr/server/solr/relationships
+#COPY --chown=solr:solr config /opt/solr/server/solr/processes
+#COPY --chown=solr:solr config /opt/solr/server/solr/collections
+#COPY --chown=solr:solr config /opt/solr/server/solr/concepts
```

Comment:
hi @ht29 should we removed the commented code? we can always compare with the `Dockerfile` in the directory.

Reply:
yes.. @outten45 .. I can remove the commented code...thanks